be/src/exprs/timezone_db.h

```diff
@@ -0,0 +1,52 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+#ifndef DORIS_BE_EXPRS_TIMEZONE_DB_H
+#define DORIS_BE_EXPRS_TIMEZONE_DB_H
+
+#include <stdint.h>
+#include <iostream>
+#include <cstddef>
+#include <sstream>
+#include <boost/date_time/posix_time/posix_time.hpp>
+#include <boost/date_time/gregorian/gregorian.hpp>
+#include <boost/date_time/time_zone_base.hpp>
+#include <boost/date_time/local_time/local_time.hpp>
+#include <boost/thread/thread.hpp>
+
+#include "common/logging.h"
+
+namespace doris {
+
+class TimezoneDatabase {
+public:
+    TimezoneDatabase();
+
+    ~TimezoneDatabase();
+
+    static void init() {
+        TimezoneDatabase();
```

Comment:
what this mean?

I think what you need is not this TimezoneDatabase constructor. 
You can move `TimezoneDatabase()` to this `init()`. 

I think there is no need to have object of this class. we only just need some static functions